andyMe/Main.jsx

```diff
@@ -8,45 +8,44 @@ const Main = () => {
     const [isEnabled, setIsEnabled] = useState(false);
     const toggleSwitch = () => setIsEnabled(previousState => !previousState);
     return(
-    <SafeAreaView>    
+    <SafeAreaView  style={[estilasos.introContenedor, { backgroundColor: isEnabled ? '#ede5ed' : '#fffbba' }]}>    
```

Comment:
After this change we don't have the empty blank space anymore.

https://github.com/Andy-Erick/My-first-app/assets/56652644/1b416732-6073-48d2-b16e-25e8ba8378a9